apps/web/src/components/content-management/care-activities/bulk-upload-modal.tsx

```diff
@@ -158,34 +143,33 @@ export const BulkUploadModalCMS: React.FC<BulkUploadModalCMSProps> = ({
           cell.protection = { locked };
         }
       }
+    }
 
-      await gapMatrixWorksheet.protect('gap-matrix-worksheet', {
-        formatColumns: true, // Allows the user to change column width and hide/unhide columns
-      });
+    // await gapMatrixWorksheet.protect('gap-matrix-worksheet', {
+    //   formatColumns: true, // Allows the user to change column width and hide/unhide columns
+    // });
```

Comment:
Commented out lines here

Reply:
I think that's the best practice. This code exists within the commit history, so if we need to have this line in future, we can find it again.